configs/example/kmh.py

```diff
@@ -41,10 +41,9 @@
     args.enable_difftest = True
     args.enable_riscv_vector = True
 
-    args.l2_hwp_type = "WorkerPrefetcher"
+    args.l2_hwp_type = "L2CompositeWithWorkerPrefetcher"
```

Comment:
Please add some comments here to tell others this does not enable CDP in kmh.py, because configs/common/CacheConfig.py turn off cdp when using kmh config